compiler/rustc_hir_typeck/src/fn_ctxt/_impl.rs

```diff
@@ -670,6 +670,8 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 | ty::PredicateKind::Clause(ty::Clause::TypeOutlives(..))
                 | ty::PredicateKind::WellFormed(..)
                 | ty::PredicateKind::ObjectSafe(..)
+                // FIXME(deferred_projection_equality): does new solver want `obligations_for_self_ty`?
```

Comment:
no ^^ this can be a `bug!` because there will never be an `AliasEq` root obligation